api/git/azure.go

```diff
@@ -25,8 +25,13 @@ const (
 )
 
 func isAzureUrl(s string) bool {
-	return strings.Contains(s, azureDevOpsHost) ||
+	condition := strings.Contains(s, azureDevOpsHost) ||
 		strings.Contains(s, visualStudioHostSuffix)
+	azureDevOpsServerHost, azureDevOpsServerHostOK := os.LookupEnv("AZURE_DEVOPS_SERVER_HOST")
```

Comment:
Comparison is against host and function name is `isAzureUrl `so wondering should we compare against url instead?

Also thinking do we even need to compare if AZURE_DEVOPS_SERVER_HOST is set.

Reply:
This function is called in [api/git/service.go](https://github.com/portainer/portainer/blob/develop/api/git/service.go) to check if a repository belongs to Azure or not. Because we can use other version control services beside Azure DevOps, the URL should always be checked to see if it's hosted on Azure or not.